.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,19 +13,55 @@ For questions, the best place to get answers is on our [discussion forum](https:
 
 Please note: We take Boundary's security and our users' trust very seriously. If you believe you have found a security issue in Boundary, please responsibly disclose by contacting us at security@hashicorp.com. Our PGP key is available at [our security page](https://www.hashicorp.com/security/).
 
+If Boundary Desktop Client version >= 2.1.0
```

Comment:
Is this "if" related to the next line about logging? What if the version is < 2.1.0?

Reply:
If the version is <2.1.0 there is no logging. They would only be able to give us the info they get from errors, along with relevant controller versions, OS version, etc.

Reply:
That clarification seems small but worth noting:

```
Logging is only available on Boundary Desktop Client version >= 2.1.0.
```

Reply:
Updated.